index.html

```diff
@@ -5224,6 +5224,79 @@ <h2>IANA Considerations</h2>
         </li>
       </ul>
 
+<section id="media-type">
+  <h2>The <code>application/credential+json</code> Media Type</h2>
+  <p>
+    This specification registers the <code>application/credential+json</code> MIME Media Type specifically for
+    identifying documents
+    conforming to the Verifiable Credentials format.
+  </p>
+  <table>
+    <tr>
+      <td>Type name: </td>
+      <td>application</td>
+    </tr>
+    <tr>
+      <td>Subtype name: </td>
+      <td>credential+json</td>
+    </tr>
+    <tr>
+      <td>Required parameters: </td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>Optional parameters: </td>
+      <td>
+        profile: The profile parameter for the application/credential+json
+        media type allows one or more profile URIs to be specified. These
+        profile URIs have the identifier semantics defined in [[!RFC6906]].
+        The "profile" media type parameter MUST be quoted. It contains a
+        non-empty list of space-separated URIs (the profile URIs).
+        <pre>profile-param = "profile=" profile-value
+profile-value = <"> profile-URI 0*( 1*SP profile-URI ) <">
+profile-URI   = URI</"></"></pre>
+        The "URI" in the above grammar refers to the "URI" as defined in
+        Section 3 of [[!RFC3986]].
+      </td>
+    </tr>
```

Comment:
```suggestion
```

It's not clear what use case this parameter is intended to address. There should probably be a section about it in the main part of the specification. It's not clear to me what a profile URI is intended to do if supplied to a VC processor.

Reply:
I'm happy to merge this change suggestion after comments from chairs.

Reply:
If no one complains about removing the section (which hasn't happened in the 3 weeks since it was suggested) and @OR13 agrees with the change, then I think it is fine to commit the suggestion.

Reply:
I support removing this optional parameter.  I've created/registered/used a lot of content types and I've never seen a parameter used.  It seems like unnecessary complexity.

Reply:
This section has not been added yet, this PR is entirely new.